### PR TITLE
Update link for waveshare e-ink crate

### PR DESCRIPTION
**Pull request by dkm**
The crate was moved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -770,7 +770,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [DS3231]: https://github.com/wose/ds3231
 [DS3234]: https://github.com/rust-embedded/wg/issues/39#issuecomment-375262785
 [DS323x]: https://crates.io/crates/ds323x
-[eink-waveshare]: https://crates.io/crates/eink_waveshare_rs
+[epd-waveshare]: https://crates.io/crates/epd-waveshare
```
Comment (therealprof) [inline README.md:773]: You will also need to update the link referring to this label, right?

Comment (dkm) [reply README.md:773]: Sorry you're right !

File: README.md [hunk 2]
```diff
@@ -684,7 +684,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 1. [DS3231] - I2C - real time clock
 1. [DS3234] - SPI - Real time clock
 1. [DS323x] - I2C/SPI - Real-time clocks (RTC): DS3231, DS3232 and DS3234 - ![crates.io](https://img.shields.io/crates/v/ds323x.svg)
-1. [eink-waveshare] - SPI - driver for E-Paper Modules from Waveshare
+1. [epd-waveshare] - SPI - driver for E-Paper Modules from Waveshare
```
Comment (therealprof) [inline README.md:687]:
```suggestion
1. [epd-waveshare] - SPI - driver for E-Paper Modules from Waveshare ![crates.io](https://img.shields.io/crates/v/epd-waveshare.svg)
```

Maybe put a crates.io version label on it while you're changing that? 😅

**Comment by dkm** [comment]
Thanks !

**Pull request merged**